### PR TITLE
do ilmerge and pack of nuget.commandline as msbuild target

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Got rid of publish-nugetexepackage cmdlet being called from build.ps1

pack and ilmerge are both now msbuild targets of nuget.commandline 

this change facilitates batched signing in VSO.

### CODE REVIEW

File: src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj [hunk 1]
```diff
@@ -84,5 +73,28 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="IlmergeNugetExe" AfterTargets="Build" Condition="'$(BuildingInsideVisualStudio)' != 'true'">
```
Comment (emgarten) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj]: Should this be before PackProjects instead?

Comment (emgarten) [reply src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj]: nit on casing: ILMergeNuGetExe

**Review by emgarten — APPROVED** [review]
Minor suggestions, overall this looks great! 🚀

File: src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj (hunk 1 above)
Comment (rohit21agrawal) [reply src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj]: i think it should be after Build. You still want the ilmerge'd nuget.exe even if someone didn't pack the project.

**Review by nkolev92 — APPROVED** [review]
🚢

**Pull request merged**